src/ray/core_worker/transport/direct_task_transport.cc

```diff
@@ -525,23 +527,11 @@ void CoreWorkerDirectTaskSubmitter::PushNormalTask(
           RAY_CHECK_GE(scheduling_key_entry.num_busy_workers, 1u);
           scheduling_key_entry.num_busy_workers--;
 
-          if (reply.worker_exiting()) {
-            RAY_LOG(DEBUG) << "Worker " << addr.worker_id
```

Comment:
The same logic is performed from `ReturnWorker` right?

Reply:
Yep.